faculty/clients/job.py

```diff
@@ -256,9 +387,47 @@ def list(self, project_id):
         endpoint = "/project/{}/job".format(project_id)
         return self._get(endpoint, JobSummarySchema(many=True))
 
-    def update_metadata(self, project_id, job_id, name, description):
+    def create_job(self, project_id, job_parameters):
```

Comment:
For consistency with other methods, I think this should be called just `create`.